main.go

```diff
@@ -66,7 +66,14 @@ func main() {
 				serviceNets = append(serviceNets, *ipnet)
 			}
 
-			operator.RunWGA(cmd.Context(), clientConfig(), serviceNets, []net.IPNet{*peersNet}, serverAddr)
+			DNSServers := os.Getenv("WGA_DNS_ADDRESSES")
+			if DNSServers == "" {
```

Comment:
wouldnt that be fine tho?

Reply:
We are the ones that set it in the status field of the CRD. 

If it's unset, then it's empty in the CRD. 

I don't think we want that because we use the CRD output in the webview to print dns spec file, or in nft to route stuff. This is 100% a config error

Reply:
i'd argue that we want to maybe support not doing dns through the tunnel at all.   but its a weak argument, and actually i cant come up with a good reason, so whatever.

Reply:
I think we should consider that use-case if it comes up, but as long as we don't have a use-case for it, it's just adding complexity to handle DNS set or unset